dashboard/src/components/SelectRepoForm/SelectRepoForm.test.tsx

```diff
@@ -16,6 +17,7 @@ const installedPackageDetail = {
   availablePackageRef: {
     context: { cluster: "default", namespace: "default" },
     identifier: "bitnami/my-chart",
+    plugin: { name: "my.plugin", version: "0.0.1" } as Plugin,
```

Comment:
Heh - yeah, so with my suggestion above, `my.plugin` would not be allowed as a name, but again, let's look for precedents in the k8s names.

Reply:
Using now `+`, but I've refactored this logic to just change the char in a single place, just in case.